site/en/blog/media-updates-in-chrome-62/index.md

```diff
@@ -0,0 +1,216 @@
+`seekable` range logic with [Media Source Extensions (MSE)] by
+providing or removing a single seekable range that is union'ed with the current
+buffered ranges. It results in a single seekable range which fits both, when
+the media source duration is `+Infinity`.
+
+In the code below, the media source has already been attached to a media
+element and contains only its init segment:
+
+```js
+const mediaSource = new MediaSource();
+...
+
+mediaSource.duration = +Infinity;
+// Seekable time ranges: { }
+// Buffered time ranges: { }
+
+mediaSource.setLiveSeekableRange(1 /* start */, 4 /* end */);
+// Seekable time ranges: { [1.000, 4.000) }
+// Buffered time ranges: { }
+
+// Let's append a media segment that starts at 3 seconds and ends at 6.
+mediaSource.sourceBuffers[0].appendBuffer(someData);
+// Seekable time ranges: { [1.000, 6.000) }
+// Buffered time ranges: { [3.000, 6.000) }
+
+mediaSource.clearLiveSeekableRange();
+// Seekable time ranges: { [0.000, 6.000) }
+// Buffered time ranges: { [3.000, 6.000) }
+```
+
+There are many cases that I didn't cover above so I'd suggest you give a try
+to [the official sample] to see how buffered and seekable time ranges react to different
+MSE events.
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/d/msg/blink-dev/-LTXhyDzS_E/LfjqN71kAAAJ) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5671401352593408) &#124;
+[Chromium Bug](https://crbug.com/623698)
+
+## FLAC in MP4 for MSE {: #flac-in-mp4-with-mse }
+
+The lossless audio coding format [FLAC] has been supported in regular media
+playback since Chrome 56. FLAC in ISO-BMFF support (aka FLAC in MP4) was added
+shortly after. And now FLAC in MP4 is available in Chrome 62 for [Media Source
+Extensions (MSE)].
+
+For info, Firefox folks are the ones who developed and implemented support for
+a [FLAC in MP4 encapsulation spec], and the BBC has been experimenting with
+using that with MSE. You can read the BBC's ["Delivering Radio 3 Concert
+Sound"] post to learn more.
+
+Here's how you can detect if FLAC in MP4 is supported for MSE:
+
+```js
+if (MediaSource.isTypeSupported('audio/mp4; codecs="flac"')) {
+  // TODO: Fetch data and feed it to a media source.
+}
+```
+
+If you want to see a full example, check out [our official sample].
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/d/msg/blink-dev/ntoLfR7rbmE/3R1DQoBSAAAJ) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5713014258925568) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=666000)
+
+## Automatic video goes to fullscreen when the device is rotated {: #auto-fullscreen-rotate }
+
+If you rotate a device to landscape while a video is playing in the viewport,
+playback will automatically switch to fullscreen mode. Rotating the device to
+portrait puts the video back to windowed mode. Check out our [past article] for
+more details.
+
+<!-- lint disable definition-case -->
```

Comment:
I'm confused - can we remove this lint thing?

Reply:
I had to include this because lint fails otherwise.